### DISCUSSION TITLE
Node-Red in Container Station Qnap?

### DISCUSSION THREAD

**Discussion by Paol8genoa** [indomus/forum#1348] [HUB: Home Assistant]
Ciao,
ho letto l'articolo sull'installazione di Node-Red su Raspy con Docker, avendo io HA su Container Station in Qnap, chiedo innanzi tutto se ha senso installarlo sul Qnap e, se si, posso utilizzare l'immagine in Doker Hub in versione latest. Grazie.

![image](https://user-images.githubusercontent.com/82047540/150180852-5f58fba2-9891-40e5-aba7-de97575539e5.png)

**Comment by snakuzzo** [comment]
Sì, sul dove metterlo devi saperlo tu. Se ti serve sul QNAP, va bene metterlo sul QNAP... e sì... puoi usare la latest
